src/commands/profile/profile.ts

```diff
@@ -435,15 +435,16 @@ const command: Command = {
           users.push(cachedUser)
         }
       } else {
-        await msg.guild?.members
-          .fetch({ query: args[1], limit: 10 })
-          .then((members) => {
-            members.forEach((member) => {
-              if (member.user.username === args[1]) {
-                users.push(member.user)
-              }
-            })
+        await msg.guild?.members.fetch().then((members) => {
+          members.forEach((member) => {
+            if (
```

Comment:
should we prioritize username over display name?
find by username first, if not found then find by display name

Reply:
priority is good, but we should show both, because user may doesn't know username or displayname of their friend